public/diagnostic.js

```diff
@@ -23,3 +23,21 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  orders:[],
+  addToCart: Ember.computed('Order', function() {
+    
+  }),
```

Comment:
It looks like you ran out of time with your `addToCart` function, tho you started correctly.